src/Symfony/Component/Translation/Translator.php

```diff
@@ -210,11 +210,14 @@ public function getFallbackLocales()
      */
     public function trans($id, array $parameters = [], $domain = null, $locale = null)
     {
+        if ('' === $id = (string) $id) {
+            return '';
```

Comment:
unrelated, but this can be considered a micro-optimization

Reply:
it fixes #33427, isn't it?

Reply:
Somehow yes, although IIUC the final solution for #33427 should be on `IntlFormatter` for 4.3 branch:
https://github.com/symfony/symfony/blob/af474ab97e9bdfe4ad9755a44850c805f16074dc/src/Symfony/Component/Translation/Formatter/IntlFormatter.php#L29